packages/lib/services/commands/ToolbarButtonUtils.ts

```diff
@@ -33,6 +33,30 @@ export default class ToolbarButtonUtils {
 		return this.service_;
 	}
 
+	// Editor commands will focus the editor after they're executed
+	private isEditorCommand(commandName: string) {
+		return (commandName.indexOf('editor.') === 0 ||
+				// These commands are grandfathered in, but in the future
+				// all editor commands should start with "editor."
+				// WARNING: Some commands such as textLink are not defined here
+				// because they are more complex and handle focus manually
```

Comment:
Thanks for adding this comment, it's always good to know that when looking at the code a few months later.

Reply:
And thanks for all your help on this!